README.md

```diff
@@ -1,2 +1,102 @@
 # ePlant
-ePlant
+ePlant is a gene-first visualization tool for plant genomes.
+
+## Using ePlant
+Create a new tab or choose a gene of interest (GOI) using the search bar on the left. 
+> Currently the only plant genome available is that of _arabidopsis thaliania_. 
```

Comment:
_Arabidopsis thaliana_ (Latin binomial nomenclature system capitalizes the genus, Arabidopsis in this case)